tests/boards/ehl_crb/gpio/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.8.2)
```

Comment:
Is there a reason this is an older version than others in the tree? I'm seeing `3.13.1` on several I spot checked just now.

Reply:
This is a typo, will update.